backend/tests/test_transforms.py

```
@@ -51,32 +53,37 @@ def test_align_datasets_with_partial_overlap(self):
 
+    @pytest.mark.unit
     def test_align_empty_dict_raises(self):
         """Empty input raises ValueError."""
         with pytest.raises(ValueError, match="No datasets provided"):
             align_datasets({})
 
+    @pytest.mark.unit
     def test_align_single_dataset(self):
         """Single DataFrame returns one-column result with all rows."""
         df = _make_df("2023-01-01", 20)
         combined, notes = align_datasets({"only": df})
```

Ruff (0.15.13)

[warning] Unpacked variable `notes` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)